src/Shared/LayoutRenderers/AspNetMvcActionLayoutRenderer.cs

```diff
@@ -20,11 +20,11 @@ namespace NLog.Web.LayoutRenderers
     /// </remarks>
     /// <example>
     /// <code lang="NLog Layout Renderer">
-    /// ${aspnet-mvc-action}    
+    /// ${aspnet-mvc-action}
     /// </code>
     /// </example>
     [LayoutRenderer("aspnet-mvc-action")]
-    public class AspNetMvcActionRenderer : AspNetMvcLayoutRendererBase
+    public class AspNetMvcActionLayoutRenderer : AspNetMvcLayoutRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.